js/ui/overlay/ui.overlay.js

```diff
@@ -303,15 +303,19 @@ const Overlay = Widget.inherit({
         this._initPositionController();
     },
 
+    _isTargetInContainer: function(container, target) {
+        return contains(container, target) || contains(container, target.getRootNode().host);
```

Comment:
Hm, this code is not related to an overlay specific => it should be placed in some common core place.
Let's try to update initial `contains` method code?